MantidPlot/src/ApplicationWindow.cpp

```diff
@@ -13625,47 +13625,47 @@ void ApplicationWindow::translateActionsStrings() {
   conestyle->setToolTip(tr("Cones"));
   conestyle->setStatusTip(tr("Cones"));
   crossHairStyle->setText(tr("Crosshairs"));
-  crossHairStyle->setMenuText(tr("Crosshairs"));
+  crossHairStyle->setText(tr("Crosshairs"));
```

Comment:
It looks as though `setText` is now called twice for each of the menu options in this section